src/types/onyx/ReportAction.ts

```diff
@@ -53,10 +56,17 @@ type Message = {
 
     /** Whether the pending transaction was reversed and didn't post to the card */
     isReversedTransaction?: boolean;
+
+    /** TODO: Only used in tests */
     whisperedTo?: number[];
+
+    /** TODO: Only used in tests */
     reactions?: Reaction[];
```

Comment:
It's used in `src/libs/ReportActionsUtils.ts`, let's add the context for these props